docs/features/multiple-networks/multi-homing.md

```diff
@@ -1,33 +1,96 @@
-network, and afterwards it iterates the list of additional attachments on the
-pod, invoking the corresponding delegate CNI implementing the logic to attach
-the pod to that particular network.
+Multihoming allows the configuration of secondary-network interfaces for K8s pods.
+OVN-Kubernetes secondary-network has three configurable topologies: layer 3, layer 2, or localnet.
+
+- A layer 3 topology is a simplification of the topology for the cluster default network - but without egress.
+- A layer 2 topology provides an isolated (no egress) cluster-wide layer2 network providing east-west traffic to the
+  cluster workloads.
+- A localnet topology is based on layer 2 topology, but also allows connecting to an existent (configured) physical
+  network to provide north-south traffic to the workloads.
+
+For layer 2 and localnet topologies, multihoming also allows IP features on secondary interfaces such as static IP
+allocation and persistent IP addresses for virtualization workloads.
 
-## Configuring secondary networks
 To allow pods to have multiple network interfaces, the user must provide the
 configurations specifying how to connect to these networks; these
-configurations are defined in a CRD named `NetworkAttachmentDefinition`.
+configurations are set in a CRD named `NetworkAttachmentDefinition`, defined by
+the [Kubernetes Network Custom Resource Definition De-facto Standard](https://github.com/k8snetworkplumbingwg/multi-net-spec).
+
+Notes:
+* layer 2 and layer 3 topologies are overlays - thus, they do not need **any** previous physical network configuration.
+
+## Prerequisites
+
+- multus-cni
+
+## Motivation
+
+Multihoming enables workloads (Pods and VMs) to use network interfaces separate from the primary network
+managed by Kubernetes. This allows them to have multiple interfaces without using complicated routing on
+the underlying physical network. This can serve for various use cases such as implementing firewall, VNF, virt, etc...
+
+Multihoming can ensure that VMs retain their IP addresses even when they move across nodes, enhancing
+workload mobility and minimizing disruptions.
+
+Multihoming is also compatible with the multi-network policy API, which can provide further security rules on the
+traffic.
+
+## User-Stories
 
-Below you will find example attachment configurations for each of the current
-topologies OVN-K allows for secondary networks.
+- As a Cluster-Admin, I want to configure secondary-networks for specific pods so that I can enable
+  specialized/sensitive workloads with distinct network requirements.
+- As a Cluster-Admin, I want to facilitate seamless live migration of VMs within so that I can maintain established TCP
+  connections and preserve VM IP configurations during migration.
+
+## User-Cases
+
+- cluster-wide overlay network on layer 2:
+  In this case example, two VMs are connected over a secondary-network, where VMA accesses a database deployed on VMB.
```

Comment:
the DB should be deployed as bare-metal on something connected to the network.

Reply:
But then it's not layer 2, but rather localnet.
hmm, I need to think of better use cases..

Reply:
think of something generic for layer2.